examples/competitive/ladderbots.json

```diff
@@ -1,7 +1,7 @@
 {
     "Bots": {
         "example_bot": {
```

Comment:
Perhaps this line could also be changed to "Put the name of the zip file here"?
I think the sc2ai.net ladder is sensitive to what name is given here.

Reply:
Good catch - I missed this.
I will do as you suggest.